integrations/coinbase.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Coinbase",
+  "auth": {
+    "authorizationURL": "https://www.coinbase.com/oauth/authorize",
+    "tokenURL": "http://www.coinbase.com/oauth/token",
+    "accessTokenURL": "https://api.coinbase.com/oauth/token",
+    "authType": "OAUTH2",
+    "tokenParams": { "grant_type": "authorization_code" },
+    "authorizationParams": { "response_type": "code" }
+  },
+  "request": {
+    "baseURL": "https://api.coinbase.com/v2/user",
```

Comment:
```suggestion
    "baseURL": "https://api.coinbase.com/v2/",
```

Reply:
but according to doc we have a value /user at end of the url, right? why is it ok to omit it?

Reply:
As far as I can tell, the Coinbase API has two different endpoints:

- https://api.coinbase.com/v2/user
- https://api.coinbase.com/v2/accounts

The `baseURL` should be the common part of these endpoints. Otherwise, only the `/user` endpoint would be accessible using Pizzly with the previous configuration.

Reply:
got it